Check personal id matches

ErikBergman opened the pull request:
The sample sorting step now has functionality to check if a personal ID for a paired sample is missing, or if there are multiple personal IDs associated with a paired sample. Tested with a small set of mockup samples, but not yet any real samples.

Bella2347 commented:
Script "KAPA_checkPairedSamplePersonalID" is missing the ending ".py"

The scripts in "99. misc", do you want to merge them? there is nothing wrong to have helper scripts for specific questions, we have other scripts like that in the folder "maintenance". 
- You could maybe move the scripts to maintenance or change the folder name to "misc" (if not the 99. is needed for something). 
- It would also be nice it you added a comment at the top of the scripts describing what they do. 

As I understand it, you first made changes to the OVARIAL_pairSamples.py script but then created a separate script for this function instead. 
In that case you would like to revert the changes to OVARIAL_pairSamples.py. You can do that by standing in your folder, in this branch, and copy the file from the live folder, the script should then have no changes compared to master. But you did do some nice renamning of variables that would be lost, if you want to keep them you have to manually change the other things back. 

Are you happy with the script or do you plan on working on it more? (I have not gone through the main script yet)

ErikBergman commented:
> Script "KAPA_checkPairedSamplePersonalID" is missing the ending ".py"
Fixed.
> The scripts in "99. misc", do you want to merge them? there is nothing wrong to have helper scripts for specific questions, we have other scripts like that in the folder "maintenance".
> 
>     * You could maybe move the scripts to maintenance or change the folder name to "misc" (if not the 99. is needed for something).
They were not needed, now removed.
>     * It would also be nice it you added a comment at the top of the scripts describing what they do.
> 
> 
> As I understand it, you first made changes to the OVARIAL_pairSamples.py script but then created a separate script for this function instead. In that case you would like to revert the changes to OVARIAL_pairSamples.py. You can do that by standing in your folder, in this branch, and copy the file from the live folder, the script should then have no changes compared to master. But you did do some nice renamning of variables that would be lost, if you want to keep them you have to manually change the other things back.
Yes, those changes were not needed, now restored.
> Are you happy with the script or do you plan on working on it more? (I have not gone through the main script yet)
It's good for now.
